src/collaboration/util.ts

```diff
@@ -38,6 +68,13 @@ export function shouldDiscardRemoteElement(localAppState: Readonly<AppState>, lo
 	) {
 		return true
 	}
+
+	// embeddables get updated a lot by the excalidraw library when collaboration is active
+	// we need to filter out useless updates to not rerender every millisecond.
+	// current master of the excalidraw library probably fixes this issue but it's not available in excalidraw@latest (0.17.6)
```

Comment:
checking if the objects are DeepEqual is a workaround. Not entirely certain, but I do think if we could use the lib like excalidraw.com does here: https://github.com/excalidraw/excalidraw/blob/master/excalidraw-app/collab/Collab.tsx#L751 we would not need this workaround. However I couldn't figure out how to use the master version of excalidraws library in our build process.

Reply:
Sounds good, being able to build from master or another branch/a fork there is something that could certainly be useful, but we can check that separately